v2/src/ctrlfbe/serializers.py

```diff
@@ -98,50 +117,47 @@ def create(self, validated_data):
 
 
 class PageDetailSerializer(serializers.ModelSerializer):
-    id = serializers.SerializerMethodField(method_name="get_id")
-    topic = serializers.SerializerMethodField(method_name="get_topic")
-    owners = serializers.SerializerMethodField(method_name="get_owners")
-    issue_id = serializers.SerializerMethodField(method_name="get_issue_id")
-
     class Meta:
-        model = PageHistory
-        exclude = ["owner", "page"]
-        read_only_fields = ["id", "created_at"]
-
-    def get_id(self, page_history):
-        temp = page_history.page.id
-        return temp
-
-    def get_topic(self, page_history):
-        temp = page_history.page.topic.id
-        return temp
-
-    def get_owners(self, page_history):
-        page = page_history.page
-        owners = list(page.owners.values_list("id", flat=True))
-        return owners
+        model = Page
 
-    def get_issue_id(self, page_history):
+    def to_representation(self, page):
+        version_no = self.context["version_no"]
+        page_history_queryset = page.page_history.filter(page=page, version_no=version_no)
+        page_history = get_object_or_404(page_history_queryset)
+        owners = serializers.PrimaryKeyRelatedField(many=True, queryset=page.owners.all())
         issue = Issue.objects.filter(related_model_id=page_history.id, related_model_type=CtrlfContentType.PAGE).first()
-        if issue is None:
-            return None
-        return issue.id
+        issue_id = issue.id if issue is not None else None
```

Comment:
[질문]

이렇게 쓰면 안되나??

```suggestion
        issue_id = issue.id if issue else None
```